etc/ci/check_no_unwrap.sh

```diff
@@ -13,4 +13,8 @@ FILES=("components/compositing/compositor.rs"
        "components/compositing/pipeline.rs"
        "components/compositing/constellation.rs")
 
-! grep -n "unwrap(\|panic!(" "${FILES[@]}"
+# make sure the files exist
+ls -1 "${FILES[@]}"
+
+# make sure the files do not contain "unwrap" or "panic!"
+! grep -Hn "unwrap(\|panic!(" "${FILES[@]}"
```

Comment:
-H is the default when there is more than one file to search, so it's redundant here.